### DISCUSSION TITLE
Change diretory prisma

### DISCUSSION THREAD

**Discussion by Heitorh3** [prisma/prisma#3046] [Q&A] [answered]
Is possible change diretory prisma for other location?

Exemple: 
to: /prisma
for: /src/provider/prisma.

**Comment by ryands17** [comment, marked as answer]
Hey @Heitorh3 👋 
Yes you can change the directory to another location. Suppose the directory is `src/provider/prisma`.

The migrate and save commands would be as follows:

```
prisma migrate save --experimental --schema ./src/provider/prisma/schema.prisma
prisma migrate up --experimental --schema ./src/provider/prisma/schema.prisma
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
